csharp/test/Microsoft.ML.OnnxRuntime.Tests.Common/OrtIoBindingAllocationTest.cs

```diff
@@ -102,6 +109,23 @@ public void TestIOBindingWithOrtAllocation()
                         Assert.Equal(outputData, tensor.ToArray<float>(), new FloatComparer());
                     }
                 }
+                // 3. Pretend we are using external allocation which is currently on CPU
```

Comment:
>Pretend we are using [](http://example.com/codeflow?start=22&length=20)

perhaps actually use a separately allocated buffer?
then there's also an example that we can refer users to.